### DISCUSSION TITLE
在Blazor项目中使用BaseEntity报错： BaseEntity.Initialization 初始化错误，获取到 IFreeSql 是 null

### DISCUSSION THREAD

**Discussion by netty2019** [dotnetcore/FreeSql#1380] [General]
在Blazor项目中使用BaseEntity报错： BaseEntity.Initialization 初始化错误，获取到 IFreeSql 是 null
![image](https://user-images.githubusercontent.com/47128297/208849479-0e63d107-ef04-4746-9472-e678d6e5cb9f.png)
![image](https://user-images.githubusercontent.com/47128297/208849575-2a2a5437-fdd6-4099-b811-ca5e779289a2.png)
![image](https://user-images.githubusercontent.com/47128297/208849719-e7b689df-71d8-4ebb-8edd-56542512e9e3.png)

**Comment by densen2014** [comment]
发demo.zip上来或者贴代码我给你看一下

**Reply by netty2019** [reply]
找到原因了，没有调初始化方法
